tests/unit/querybuf.tcl

```diff
@@ -34,10 +34,15 @@ start_server {tags {"querybuf slow"}} {
             fail "query buffer was resized"
         }
 
+        puts [client_query_buffer test_client]
+
         # Fill query buffer to more than 32k
         $rd set bigstring v ;# create bigstring in advance to avoid adding extra memory
         $rd set bigstring [string repeat A 32768] nx
```

Comment:
```suggestion
        $rd set bigstring [string repeat A 32767] nx
```
This allows the querybuf to be expanded to 64k, and prevents the querybuf from being reset to NULL.